cmd/traefik/traefik.go

```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/containous/traefik/types"
 	"github.com/containous/traefik/version"
 	"github.com/coreos/go-systemd/daemon"
+	"github.com/ogier/pflag"
```

Comment:
FWIW, most people use spf13/pflag instead of ogier's.

Reply:
come from flaeg:

https://github.com/containous/flaeg/blob/c4797734e8b31bd78dc2cc5d88d510decdee2d7e/Gopkg.toml#L25

Reply:
All good then, thanks. :+1: